### ISSUE TITLE
FieldTag definition

### ISSUE THREAD

**Issue by cKlee** [MARCspec/MARCspec#26]
With respect to http://www.loc.gov/marc/specifications/specrecstruc.html#varitags field tags in ANSI Z39.2 and ISO 2709 could consist of both alphabetic and numeric characters, although MARC 21 formats use only numeric tags.

The current Spec embraces this possibility by:

    fieldTag  = 3(alphalower / DIGIT / ".") / 3(alphaupper / DIGIT / ".")

@pkiraly suggests to disallow alphabetic characters and make `LDR` or `LEADER` an explicit field tag:

    fieldTag  = 3(DIGIT / ".") /  "LDR" / "LEADER"

But if the overall MARCspec should cope ANSI Z39.2 and ISO 2709, we should support it, if this does not cause problems.

About the leader: `LDR` is already covered by `3(DIGIT)`, so why make this explicit? And does `LEADER` actually appear in data? This usage might lead to additional efforts for parsing. Is it necessary?

**Comment by pkiraly** [comment]
I haven't checked ANSI Z39.2 and ISO 2709 but seems MARC is a subset. So it is not allowed tags like "2a3".

**Comment by pkiraly** [comment]
I haven't checked ANSI Z39.2 and ISO 2709 but seems MARC is a subset. So it is not allowed tags like "2a3".

**Comment by cKlee** [comment]
@pkiraly Yes, MARC is a subset and more restrictive than ANSI/ISO. So the question is, should MARCspec support 'all' implementations of ANSI/ISO?

**Comment by danizen** [comment]
What are the implementations outside of library technology?   I had been thinking to suggest that the valid tag values be amended so that only LDR and numeric (including '.' formulations) be recognized):

```abnf
fieldTag          = 3(DIGIT / ".") / "LDR"
```

However, it is easy enough to add that in semantics.
